### ISSUE TITLE
Etiketten für mehrere Dateien gleichzeitig setzen

### ISSUE THREAD

**Issue by LazyGuy21No** [jlawyerorg/j-lawyer-org#1629]
Aktuell kann man über die Mehrfachauswahl (wohl) keine Etiketten 🏷 setzen. Es wäre schön, wenn das ginge.

**Comment by Dickmaulruessler** [comment]
Ja, die Möglichkeit, die Etiketten für **mehrere** Dokumente gleichzeitig bearbeiten zu können, würde ich mir auch wünschen - allerdings nicht Etiketten "setzen", sondern "**löschen**". 

Mein häufigster Anwendungsfall ist, dass ich im Scaneingang bzw. Posteingang alle neuen Dokumente mit dem Etikett "NEU" markiere und diese Etiketten nach Sichtung und Weiterbearbeitung in der Akte lösche. 
Wenn man dann sehr umfangreiche Eingänge hat, klickt man sich zu Tode (gerade klicke ich mich einzeln durch 86 Dokumente und lösche jeweils einzeln das Etikett "NEU"...). 
Ja OK, 86 Dokumente sind sicherlich nicht der Normalfall, 5-10 Dokumente aber durchaus schon.

**Comment by Dickmaulruessler** [comment]
#1632
